controllers/openstackcluster_controller.go

```diff
@@ -174,31 +174,41 @@ func (r *OpenStackClusterReconciler) reconcileDelete(ctx context.Context, scope
 		}
 
 		if err = loadBalancerService.DeleteLoadBalancer(openStackCluster, clusterResourceName); err != nil {
-			handleUpdateOSCError(openStackCluster, fmt.Errorf("failed to delete load balancer: %w", err))
+			if !openStackCluster.Status.Ready {
+				handleUpdateOSCError(openStackCluster, fmt.Errorf("failed to delete load balancer: %w", err))
+			}
```

Comment:
Don't we want it stored in events though?

Reply:
Ah nevermind this is not for events. This is the last error.
I agree this can't be the last error because other resources underneath can't be removed either if this one wasn't (hopefully my reasoning makes sense).